### ISSUE TITLE
Spellpower Set Bonus Bug

### ISSUE THREAD

**Issue by Oudeis1447** [elysium-project/server#98]
I have found a bug with spellpower increase Set Bonus (at least Arcanist (mage T1)).

First thing to know, you can you your spellpower of a specific school on your character sheet if you mouse over your Ranged Attack damage. The +xx damage is the amount of SP of the school of your wand. If your wand do arcane damage, the +xx is your +spellpower in arcane. I have tested this with arcane missile since it's the spell with the most consistent damage (always do 200 a tick at max rank in a arcane spec, then about +1 each 5 SP), and the damage always fit with the amount of SP shown on the Ranged Attack damage.

My character start with 310 Arcane Spell Power (SP). By manually adding my +SP and Arcane SP, I have calculated that is exactly what I should have. If I remove one piece of the set(set giving +18 SP), my character sheet show that I am at 250, loose 60 SP. The item I removed was arcanist binding, so +6 SP with the +18 I should only loss 24SP, so I loose 36 SP for unknown reason. If I put back the bracers, I go back to 274 so I get everything I should from the bracer but I still miss 36SP. By removing parts of my gear, I have found out that while missing those 36 SP, if I remove my Strompike Cloth Girdle and my Songstone of Ironforge, both +18 SP item,  I don't loose any SP removing them and I get 18 SP each if put them back on, bringing me back to 310 SP. +18 SP item, +18 Set bonus, I think I have found the issue.

TLDR: by loosing a SP set bonus, you loose all SP from item with the same amount of SP.

**Comment by thepparker** [comment]
This is a bug with all set bonuses and is described better here: https://elysium-project.org/bugtracker/issue/851
